src/CMakeLists.txt

```diff
@@ -863,6 +863,9 @@ add_executable(ceph-osd ${ceph_osd_srcs})
 add_dependencies(ceph-osd erasure_code_plugins)
 target_link_libraries(ceph-osd osd os global-static common
   ${BLKID_LIBRARIES} ${RDMA_LIBRARIES})
```

Comment:
here, if just link to common (common link to ceph-common-deps), can't link all DPDK_LIBRARIES (don't know why), but do  target link here again it works.